core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -346,26 +346,26 @@
   .oppia-contributor-dashboard-collapsible-header {
     display: none;
   }
-  .oppia-fade-in {
-      -webkit-animation: fadein 0.5s ease-out alternate;
-      -moz-animation: fadein 0.5s ease-out alternate;
-      animation: fadein 0.5s ease-out alternate;
+  .oppia-translate-y {
+      -webkit-animation: translate-y 0.5s ease-out alternate;
+      -moz-animation: translate-y 0.5s ease-out alternate;
+      animation: translate-y 0.5s ease-out alternate;
   }
   .oppia-language-selector {
     padding-bottom: 10px;
   }
   .oppia-collapsed-language-selector {
     padding-bottom: 15px;
   }
-  @-webkit-keyframes fadein {
```

Comment:
This is not fade-in, it drops the element from top to down. Should I change the name to `drop` or `translate-y` is fine?

[Also, this was causing regression issue as we have another fadein animation in the contributor dashboard which was using this style instead of the original/expected one.]